src/styles/components/PostPage.modules.css

```diff
@@ -0,0 +1,121 @@
+/* General Styles */
+body {
```

Comment:
We are using Bootstrap for styling to avoid writing our own CSS. CSS is really hard to debug and normalize so we can use a library to handle all of that for us.